tests/manage/pv_services/test_resource_deletion_during_pod_pvc_deletion.py

```diff
@@ -0,0 +1,323 @@
+        final_num_resource_to_delete = len(pod_functions[resource_to_delete]())
+        assert final_num_resource_to_delete == num_of_resource_to_delete, (
+            f"Total number of {resource_to_delete} pods is not matching with "
+            f"initial value. Total number of pods before deleting a pod: "
+            f"{num_of_resource_to_delete}. Total number of pods present now: "
+            f"{final_num_resource_to_delete}"
+        )
+
+        # Check ceph status
+        ceph_health_check(namespace=config.ENV_DATA['cluster_namespace'])
+        log.info("Ceph cluster health is OK")
+
+
+@tier4
+@pytest.mark.parametrize(
+    argnames=['interface', 'operation_to_disrupt', 'resource_to_delete'],
+    argvalues=[
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL, 'delete_pvcs', 'mgr'],
+            marks=pytest.mark.polarion_id("OCS-922")
+        ),
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL, 'delete_pods', 'mgr'],
+            marks=pytest.mark.polarion_id("OCS-923")
+        ),
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL, 'delete_pvcs', 'mon'],
+            marks=pytest.mark.polarion_id("OCS-914")
+        ),
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL, 'delete_pods', 'mon'],
+            marks=pytest.mark.polarion_id("OCS-911")
+        ),
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL, 'delete_pvcs', 'osd'],
+            marks=pytest.mark.polarion_id("OCS-912")
+        ),
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL, 'delete_pods', 'osd'],
+            marks=pytest.mark.polarion_id("OCS-913")
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM, 'delete_pvcs', 'mgr'],
+            marks=pytest.mark.polarion_id("OCS-920")
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM, 'delete_pods', 'mgr'],
+            marks=pytest.mark.polarion_id("OCS-915")
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM, 'delete_pvcs', 'mon'],
+            marks=pytest.mark.polarion_id("OCS-918")
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM, 'delete_pods', 'mon'],
+            marks=pytest.mark.polarion_id("OCS-919")
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM, 'delete_pvcs', 'osd'],
+            marks=pytest.mark.polarion_id("OCS-924")
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM, 'delete_pods', 'osd'],
+            marks=pytest.mark.polarion_id("OCS-917")
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM, 'delete_pvcs', 'mds'],
+            marks=pytest.mark.polarion_id("OCS-916")
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM, 'delete_pods', 'mds'],
+            marks=pytest.mark.polarion_id("OCS-921")
+        )
+    ]
+)
+class TestDeleteResourceDuringPodPvcDeletion(DisruptionBase):
```

Comment:
This class inherits functionality form DisruptionBase class. pvc_objs, sc_obj and namespace are set using this feature. interface parameter is required in both 'setup_base' and 'test_disruptive_during_pod_pvc_deletion' methods.
If the test class is removed, whole code has to be re-structured. Also, it will be difficult to add more test methods in future.